healthcheck/urls.py

```diff
@@ -2,13 +2,18 @@
 from django.urls import include, path  # noqa: F401
 from rest_framework import routers
 
+from selfswab.urls import v2router as selfswabscreen_v2router
```

Comment:
Please rename `selfswabscreen_v2router` to `selfswabs_v2router` and remove the commented out lines